jax_triton/triton_lib.py

```diff
@@ -89,6 +90,11 @@
     jnp.dtype("bool"): "B",
 }
 
+def is_device_rocm():
```

Comment:
This code is broken. The lowering rule is told the platform it is targeting, and code should be emitted for that platform. You should not call runtime APIs here.

Reply:
@hawkinsp  @superbobry I was doing to determine the platform. If I don't do this, then what's a good alternative to do that? 
Thanks!

Reply:
I think the best way to do this is to register two platform-specific lowering rules, one for cuda, one for rocm (via the `platform="..."` argument to `register_lowering_rule`). You can then pass a bool or a string to the lowering rule so it knows which it is targeting.